src/_posts/platform/getting-started/2000-01-01-setup-ssh-macos.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Setup SSH on Mac OS
+title: Setup SSH on macOS
```

Comment:
If you want to change the title of a page, you probably also need to add a redirection from the old to the new URL in `redirections.yml`